### ISSUE TITLE
GT 변환 제대로 됬는지 확인

### ISSUE THREAD

**Issue by ssaru** [DeepBaksuVision/YOLOv1#12]

**Comment by ssaru** [comment]
`import torchvision.transforms as transforms`의 `transforms.ToTensor()`를 dataloader에 transform 인자로 주면 다음과 같이 이미지 변환이 됨.

원하지 않았던 결과로 변환

![before](https://user-images.githubusercontent.com/13328380/45957180-612ba600-c04f-11e8-8755-5826b51ececa.png)
![after](https://user-images.githubusercontent.com/13328380/45957184-62f56980-c04f-11e8-93ca-06bd8dce239b.png)

**Comment by ssaru** [comment]
마지막 feature map 고려해서 7x7 grid image로 label과 매칭시, label matching이 이상하게 표현됨. 
xml 기반의 labeling matching을 확인해봐야할 것 같음

![gridimage](https://user-images.githubusercontent.com/13328380/45957337-bebff280-c04f-11e8-8bee-fb003e8ecbf6.png)

**Comment by ssaru** [comment]
xml을 이용해 yolo format으로 변환된 label을 복원해서 label이 image와 잘 매칭이 되는지 확인했으나,

7x7 grid에 맞춰서 변환한 결과가 grid cell 기준 x축 y축 각각 1씩 차이나 나는것으로 판단됨

![origin_label](https://user-images.githubusercontent.com/13328380/45963569-693f1200-c05e-11e8-9ab3-b1d925df3e4b.png)
![convert_label](https://user-images.githubusercontent.com/13328380/45963575-6b08d580-c05e-11e8-8188-48f5fbd30a2b.png)

**Comment by insurgent92** [comment]
완벽하군요!!
